data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1103 @@
+			`	"Hail, captain <last>!" Riijaaora says, this time speaking with their own voice. The sounds feel warped and uneven, as if they are being rearranged midair. "We see that thou returns from thy hunt successful!"`
+			choice
+				`	"Yes, I have."`
+				`	"Of course."`
+			`	Riijaaora wriggles their body in what might be an expression of joy before they and the other scions splay their arms before you. "Thou hast done this House great service, captain," they say. "We would be proud to call thee 'ally,' and reward thee for thine usefulness." One of the military officers standing beside the scions moves towards you and gently takes the caged animal from your hands. It chirps quietly as you hand it over.`
+			`	Riijaaora gestures to a small wooden pin affixed to the outer layer of their clothing. "We keep a number of these devices, full of sound and language, but very few for thine own tongue. This one... 'tis an art piece, issued many years past for a performance of thine own Shakespeare's Tempest so that our noble Predecessors could experience the work in its truest form. It is now thy promised gift."`
+			`	Riijaaora plucks the device from their outfit and offers it to you.`
+			choice
+				`	(Take it.)`
+			`	The device is made of carved wood, or at least something that appears like it, depicting in well-varnished grain a sailing ship washed against rocks. As you pin it on your jacket, the ship begins to gently rock in place. Riijaaora turns away from you to speak to Sieasej. "And think not, fair citizen, that thine own contributions shall go without recognition. Thou shalt be rewarded, too, for thy part in this." Riijaaora gestures for another officer to escort them away, though not before Sieasej offers you a short splay of the arms in goodbye.`
+			`	An officer approaches you to offer you a data-sphere. "Within this," Riijaaora says, "lies knowledge of the history of most worlds within our space. Combined with thy newfound skill of tongues, thou should'st find our worlds and cities more revealing to thy gaze and more fruitful for thine efforts. Thou shall'st be permitted to looks upon our creations and our vessels. Should'st the prospect interest thee, we're sure that the other Houses in their council would agree to extend thee purchase rights, provided thou displays to them thy beneficence and utility. There're many minor tasks which need completion on our worlds; no doubt thy prompt assistance would help in such ingratiation to the others." Riijaaora wiggles. "'Tis all, unless thou hast more questions thou needeth us t'resolve?"`
```

Comment:
Does Riijaaora have a second device to speak with?

Reply:
No. The intention for this section is that once First Last takes the device, it now functions to translate all the native Successor language they hear (which Riijaaora is speaking) to FL's own language. I could make this clearer with a sensory description of surrounding sounds feeling "different" once FL takes it, e.g. `...rock in place, and you feel a gentle tingling in your ear canals.` or something like that.

Reply:
Ah, that makes sense.